sky/serve/controller.py

```diff
@@ -126,6 +126,21 @@ async def update_service(request: fastapi.Request):
                              f'{common_utils.format_exception(e)}')
                 return {'message': 'Error'}
 
+        @self._app.post('/controller/terminate_replica')
+        async def terminate_replica(request: fastapi.Request):
+            request_data = await request.json()
+            try:
+                replica_id = request_data.get('replica_id')
+                if replica_id is None:
+                    return {'message': 'Error: Replica ID is not specified.'}
+                logger.info(f'Terminating replica {replica_id}')
+                self._replica_manager.scale_down(replica_id)
+                return {'message': 'Success'}
+            except Exception as e:  # pylint: disable=broad-except
+                logger.error(f'Error in terminate_replica: '
+                             f'{common_utils.format_exception(e)}')
+                return {'message': 'Error'}
```

Comment:
we should include error message to the return value as well, so the user could see it in their local laptop (instead of doing `sky serve logs --controller <service-name>`)